invenio_github/api.py

```diff
@@ -107,6 +99,30 @@ def session_token(self):
     )
     """Return OAuth remote application."""
 
+    def check_repo_access_permissions(self, repo):
+        """Checks permissions from user on repo.
```

Comment:
After discussing with @slint , we decided to change how the permissions on a repo are computed. 

Namely, an user can read / enable / disable / hook a repo if the user is the owner OR has admin access in GitHub.